### ISSUE TITLE
Mixpanel does not comply with PSR-4

### ISSUE THREAD

**Issue by thomasedwards** [mixpanel/mixpanel-php#59]
Mixpanel should be in its own namespace in order to comply:

https://www.php-fig.org/psr/psr-4/

> A fully qualified class name has the following form:
>
> `\<NamespaceName>(\<SubNamespaceNames>)*\<ClassName>`
>
> - The fully qualified class name MUST have a top-level namespace name, also known as a “vendor namespace”.
> - The fully qualified class name MAY have one or more sub-namespace names.
> - The fully qualified class name MUST have a terminating class name.
> - Underscores have no special meaning in any portion of the fully qualified class name.
> - Alphabetic characters in the fully qualified class name MAY be any combination of lower case and upper case.
> - All class names MUST be referenced in a case-sensitive fashion.

**Comment by maxleroy** [comment]
Yes, it would be highly appreciated if Mixpanel would update the packages to use "modern technologies" such as namespaces (php5.3 😬) and psr-4 autoloading, may be even provide a class to help testing (like a handler that can be inspected to use in functional tests..)

**Comment by Rockburner** [comment]
Seconded.  Trying to implement mixpanel now and this is a definite impediment to an easy implementation.

**Comment by mauriblint** [comment]
Same problem here
